app/js/lib/simple_currency_formatter.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * This number formatter class is a primitive subset of the intl library ( https://www.npmjs.com/package/intl ),
+ * which is not available in all browsers and node versions.
```

Comment:
Is this something you wrote or something copied from elsewhere? If the later, we should probably specify from where, and what license it has.